department-of-veterans-affairs/caseflow
# Show Transcription task instructions and add Assigned By column

Resolves [CASEFLOW-2593](https://vajira.max.gov/browse/CASEFLOW-2593)

### Description
This adds a tooltip to Transcription tasks in the "Tasks" column of the Transcription Team's Unassigned tab table. It also adds an "Assigned By" column to that table, as we learned that would be helpful for the way they work their cases. 

### Testing Plan
1. Log in as the TRANSCRIPTION_USER
2. Open Queue
3. Switch your view to "Transcription team cases." 
4. If you do not see any transcription tasks, you may need to create one in the Rails console, and give it these values:
`assigned_to_type = "Organization"`
`assigned_to_id = Organization.find_by_name("Transcription").id`
`parent = AssignHearingDispositionTask.last  (and from the result of that, copy the appeal_id)`
`appeal = Appeal.find(copied_appeal_id)`
`instructions = ["Sample instruction"]`
5. Save it, and refresh the table. 

### User Facing Changes
![TranscriptionTaskTooltip](https://user-images.githubusercontent.com/92495948/142704202-e001e235-b5bc-4219-9cae-e8f0d7210b32.gif)
